packages/core/src/components/tag/tag.tsx

```diff
@@ -99,8 +99,9 @@ export const Tag: React.FC<TagProps> = React.forwardRef((props, ref) => {
     } = props;
 
     const isRemovable = Utils.isFunction(onRemove);
+    const isInterative = interactive || htmlProps.onClick != null;
```

Comment:
It might be useful to allow someone to still explicitly set `interactive: false` even when they provide an `onClick` handler, in case that's behavior they're already relying on for some reason.

To do that, should we change this to `interactive ?? htmlProps.onClick != null` and remove the default `= false` on line 89 above?

Reply:
Yeah good call

Reply:
should this `interactive ?? htmlProps.onClick != null` defaulting behavior happen in `useInteractiveAttributes`? right now I think every consumer of this hook would need to do something similar

and maybe it should be a `disabled` option instead? if a component is using this hook I think we should infer it's interactive based on being provided an `onClick`, then `disabled` can be offered as a feature so a consumer doesn't have to do something like conditionally provide `onClick`

it seems like we ended up with `interactive` as an option to this hook since the `Tag` originally had it, but now we're walking back on `interactive` needing to be set to get the interactive attributes